jobs/JGLOBAL_LAND_ANALYSIS_FINALIZE

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATA=${DATA:-${DATAROOT}/${RUN}landanl_${cyc}}
+source "${HOMEgfs}/ush/jjob_header.sh" -e "landanlfinal" -c "base landanl landanlfinal"
+
+##############################################
+# Set variables used in the script
+##############################################
+export CDATE=${CDATE:-${PDY}${cyc}}
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
+export COMPONENT="atmos"
```

Comment:
We're trending away from using this because scripts are generally geared towards a specific component anyway. Please remove these and replace with hard-coded `atmos`.

Reply:
@WalterKolczynski-NOAA As you suggested, I removed the `COMPONENT` from the script and replaced with hard-coded `atmos`. Thanks again.

Reply:
Thanks for updating this file. The other j-jobs also need the same update.

Reply:
Thanks @WalterKolczynski-NOAA and I have updated all j-jobs.